### DISCUSSION TITLE
Linkerd2 datadog integration

### DISCUSSION THREAD

**Discussion by srinath-panda** [linkerd/linkerd2#9014] [General] [answered]
new to Linkerd, is there any example for integrating linkerd with datadog. 
Wanted to use datadog instead of prometheus?

**Comment by olix0r** [comment, marked as answer]
Good question! I know that there are definitely people using Linkerd and Datadog (probably by having datadog collect Linkerd's prometheus metrics), but I'm not sure whether Datadog has out-of-the-box integrations. That's probably a better question for Datadog :)

**Comment by adleong** [comment]
We don't have any examples of this, but it looks like Datadog may be able to collect from the prometheus-style metrics that the Linkerd proxy exposes: https://docs.datadoghq.com/containers/kubernetes/prometheus
